pulp_ansible/app/migrations/0048_merge_20230223_1246.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.18 on 2023-02-23 12:46
```

Comment:
I prefer to keep the migration history linear. Sorry.
Renaming the file and switching the dependency lines works usually. But you also added no manual part, so just deleting and recreating may be easier.